libs/langchain/langchain/agents/agent_iterator.py

```diff
@@ -101,57 +83,16 @@ def inputs(self) -> dict[str, str]:
     def inputs(self, inputs: Any) -> None:
         self._inputs = self.agent_executor.prep_inputs(inputs)
 
-    @property
-    def callbacks(self) -> Callbacks:
-        return self._callbacks
-
-    @callbacks.setter
-    @rebuild_callback_manager_on_set
-    def callbacks(self, callbacks: Callbacks) -> None:
-        """When callbacks are changed after __init__, rebuild callback mgr"""
-        self._callbacks = callbacks
-
-    @property
-    def tags(self) -> Optional[List[str]]:
-        return self._tags
-
-    @tags.setter
-    @rebuild_callback_manager_on_set
-    def tags(self, tags: Optional[List[str]]) -> None:
-        """When tags are changed after __init__, rebuild callback mgr"""
-        self._tags = tags
-
     @property
     def agent_executor(self) -> AgentExecutor:
         return self._agent_executor
 
     @agent_executor.setter
-    @rebuild_callback_manager_on_set
     def agent_executor(self, agent_executor: AgentExecutor) -> None:
         self._agent_executor = agent_executor
         # force re-prep inputs in case agent_executor's prep_inputs fn changed
         self.inputs = self.inputs
 
-    @property
```

Comment:
Code is definitely better without all the helper methods here -- it's a breaking change, but I think OK -- none of those methods should be here